### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by LorenzMeier**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```
Comment (hamishwillee) [inline CODE_OF_CONDUCT.md]: Either Contributor Code of Conduct, or Contributor Covenant, not both.

**Comment by DonLakeFlyer** [comment]
Does this show up on the web site somewhere? Or just in the file?

**Comment by hamishwillee** [comment]
Good point, it should perhaps be in the user or devguide, but should certainly be linked.

**Comment by DonLakeFlyer** [comment]
Yeah, just a link I think

**Comment by hamishwillee** [comment]
Linked inhttps://github.com/mavlink/qgc-dev-guide/pull/40

**Pull request merged**